lib/msf/core/handler/reverse_tcp_double.rb

```diff
@@ -95,16 +95,22 @@ def start_handler
       sock_inp = nil
       sock_out = nil
 
-      print_status("Started reverse double handler")
+      unless datastore['MODULEOWNER'] == Msf::Exploit::Remote::BrowserAutopwnv2
```

Comment:
I kind of just followed what the old one is doing. I can change this to module.owner, I didn't know this one.